apps/package.json

```diff
@@ -229,6 +229,8 @@
     "@code-dot-org/js-interpreter": "^1.3.9",
     "@code-dot-org/snack-sdk": "2.2.0-apkurl-v1",
     "crypto-js": "^3.1.9-1",
+    "d3-geo": "^1.11.6",
+    "d3-scale": "^3.0.0",
```

Comment:
Woops! That should actually be removed now that we are simply plotting the same size dot for each map coordinate. If there is a plan to change the size of the dot based on certain criteria then it would be useful to keep but for this PR, it makes sense to remove it so I'll go ahead and do that! Thanks for catching that!